src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -777,14 +797,13 @@ namespace InternalHeatGains {
                                 } else if (mrtType == "SURFACEWEIGHTED") {
                                     People(Loop).MRTCalcType = SurfaceWeighted;
                                     People(Loop).SurfacePtr = UtilityRoutines::FindItemInList(AlphaName(8), Surface);
-                                    if (People(Loop).SurfacePtr == 0 && (People(Loop).Fanger || People(Loop).Pierce || People(Loop).KSU)) {
+                                    if (People(Loop).SurfacePtr == 0 && ThermalComfortRuns) {
                                         if (Item1 == 1) {
                                             ShowSevereError(state, RoutineName + CurrentModuleObject + "=\"" + AlphaName(1) + "\", " + cAlphaFieldNames(7) +
                                                             '=' + AlphaName(7) + " invalid Surface Name=" + AlphaName(8));
                                             ErrorsFound = true;
                                         }
-                                    } else if (Surface(People(Loop).SurfacePtr).Zone != People(Loop).ZonePtr &&
-                                               (People(Loop).Fanger || People(Loop).Pierce || People(Loop).KSU)) {
+                                    } else if (Surface(People(Loop).SurfacePtr).Zone != People(Loop).ZonePtr && ThermalComfortRuns) {
                                         ShowSevereError(state, RoutineName + CurrentModuleObject + "=\"" + AlphaName(1) + "\", Surface referenced in " +
                                                         cAlphaFieldNames(7) + '=' + AlphaName(8) + " in different zone.");
```

Comment:
This looks wrong `cAlphaFieldNames(7) + '=' + AlphaName(8)` should both be 7 or 8, I think.

Reply:
This is fixed.